traffic_ops/traffic_ops_golang/server/servers_update_status.go

```diff
@@ -97,29 +104,31 @@ SELECT
 	use_reval_pending.value,
 	s.upd_pending,
 	status.name AS status,
-		/* True if the cachegroup parent or any ancestor topology node has pending updates. */
-		TRUE IN (
-			SELECT sta.upd_pending FROM server_topology_ancestors sta WHERE sta.base_server_id = s.id
-			UNION SELECT COALESCE(BOOL_OR(ps.upd_pending), FALSE)
-		) AS parent_upd_pending,
-		/* True if the cachegroup parent or any ancestor topology node has pending revalidation. */
-		TRUE IN (
-			SELECT sta.reval_pending FROM server_topology_ancestors sta WHERE sta.base_server_id = s.id
-			UNION SELECT COALESCE(BOOL_OR(ps.reval_pending), FALSE)
-		) AS parent_reval_pending
+	/* True if the cachegroup parent or any ancestor topology node has pending updates. */
+	TRUE IN (
+		SELECT sta.upd_pending FROM server_topology_ancestors sta WHERE sta.base_server_id = s.id
+		UNION SELECT COALESCE(BOOL_OR(ps.upd_pending), FALSE)
+	) AS parent_upd_pending,
+	/* True if the cachegroup parent or any ancestor topology node has pending revalidation. */
+	TRUE IN (
+		SELECT sta.reval_pending FROM server_topology_ancestors sta WHERE sta.base_server_id = s.id
+		UNION SELECT COALESCE(BOOL_OR(ps.reval_pending), FALSE)
+	) AS parent_reval_pending
 	FROM use_reval_pending,
 		 server s
 LEFT JOIN status ON s.status = status.id
 LEFT JOIN cachegroup cg ON s.cachegroup = cg.id
 LEFT JOIN type ON type.id = s.type
 LEFT JOIN parentservers ps ON ps.cachegroup = cg.parent_cachegroup_id
 	AND ps.cdn_id = s.cdn_id
-WHERE s.host_name = $4
+WHERE s.host_name = $5
 GROUP BY s.id, s.host_name, type.name, server_reval_pending, use_reval_pending.value, s.upd_pending, status.name
 ORDER BY s.id
 `
 
-	rows, err := tx.Query(selectQuery, tc.CacheStatusOffline, tc.UseRevalPendingParameterName, tc.GlobalConfigFileName, hostName)
+	cacheStatusesToCheck := []tc.CacheStatus{tc.CacheStatusOnline, tc.CacheStatusReported}
```

Comment:
I think we may also want to consider `ADMIN_DOWN`, since that is the existing behavior today. Thinking about revalidations, we'd still want to have an `ADMIN_DOWN` mid revalidate before its children, otherwise when it comes back up it could serve invalidated content to the edges still.

Reply:
Considering `ADMIN_DOWN` in 03c22553ad